### PR TITLE
add support for manifest.yml file in git

### PR DESCRIPTION
**Pull request by ramaboggarapu**

### CODE REVIEW

File: src/messages/en.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
    "github.deploy.app.complete": "Deployment of application *%s* is complete and the app has been started.",
    "github.deploy.app.unknown": "Deployment status of application *%s* is unknown at this time. Check the status of the application and logs for further information.",
    "github.deploy.error": "A deployment error occurred for app *%s*: %s.",
+	 "github.deploy.manifest.error": "Invalid manifest.yml file for app *%s*.",
```
Comment (clanzen) [inline src/messages/en.json]: Minor - but could you clean up the indentation here?  If its like the issue I had with my Atom editor, you need to use spaces to indent.  I turned on a feature there where I can see the space more explicitly, which helps long term.

File: src/scripts/deploy.js [hunk 2]
```diff
@@ -340,6 +385,51 @@ const deploy = (app, appGuid, spaceGuid, spaceName, robot, res) => {
...
+										return true;
+									}
+								});
+								if (!found) {
+									let routeOptions = {
+										host: applicationHost,
+										domain_guid: applicationDomainGuid,
+										space_guid: spaceGuid};
+									cf.Routes.add(routeOptions).then((result) => {
+										console.log('Route Creation Result' + JSON.stringify(result));
```
Comment (clanzen) [inline src/scripts/deploy.js]: I'm guessing you want this to be a robot.logger call.

**Comment by ramaboggarapu** [comment]
@clanzen Thanks for the review. Addressed issues as per your comments.

**Comment by clanzen** [comment]
👍

**Pull request merged**